src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindOperationWithSortIntegrationTest.java

```diff
@@ -115,12 +111,14 @@ public void sortWithSkipLimit() {
           .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
           .then()
           .statusCode(200)
+          .body("status", is(nullValue()))
+          .body("errors", is(nullValue()))
+          .body("data.documents", hasSize(10))
           .body("data.documents", jsonEquals(arrayNode.toString()));
```

Comment:
Can we also test with sorting by `{ username: -1 }` just to make sure sort is being applied? AFAIK this test would pass even if sort option was ignored.

Reply:
Hmm, there should be a test for desc as well.. Will re-check..

Reply:
Yes there is `sortDescendingTextValue` with `-1`